README.md

```diff
@@ -24,6 +24,8 @@ Platform | Name |Description
 `sensor` | `heart_points_daily` | [Heart Points][heart-points] earned. Reset daily.
 `sensor` | `height` | [Height][height] (metres).
 `sensor` | `weight` | [Weight][weight] (kilograms).
+`sensor` | `fat` | [Body Fat][fat] (percentage).

```

Comment:
```suggestion
`sensor` | `body_fat` | [Body Fat][fat] (percentage).
```
The sensor name should match the automatic name Home Assistant gives it which is lowercase, snakecase